### ISSUE TITLE
Pandoc export to ~/Desktop fails on macOS Catalina

### ISSUE THREAD

**Issue by johnblommers** [vkbo/novelWriter#99]
This is a slightly different problem from [issue 92](https://github.com/vkbo/novelWriter/issues/92). In this case when the user selects a Pandoc export and the destination is

```
~/Desktop/Dick_and_Jane.tmp
```

then novelWriter complains in the console and with a popup message:

```
ERROR     Failed to convert file using pypandoc + Pandoc.
ERROR     source_file is not a valid path
```

Note that the file `~/Desktop/Dick_and_Jane.tmp` is created on the Desktop and contains the expected HTML code. Indeed the user is prompted for permission to overwrite it on subsequent export to Pandoc attempts for the same project.

**Comment by vkbo** [comment]
Ah, you're right. The Pandoc export reads the path again from the text box, which bypasses the expandusaer call. That is unnecessary. It's a simple fix.

**Comment by vkbo** [comment]
Fixed in PR #104.
